ide-plugins.md

```diff
@@ -0,0 +1,34 @@
+# IDE plugins configuration required
+
+## Prettier (automatic formatting code)
+
+If you're using VS Code, install the **_Prettier - Code formatter_** plugin ([for VS Code](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode")).
+
+Next, create a **.prettierrc** file to set basic settings. Our basic settings:
+
+```json
+{
+  "semi": true,
+  "singleQuote": true,
+  "jsxSingleQuote": false,
+  "bracketSpacing": true,
+  "trailingComma": "all",
+  "printWidth": 100,
+  "tabWidth": 2
+}
+```
+
+You can find more info about Prettier settings at the [link](https://prettier.io/docs/en/options.html).
+
+Also you can use a package Prettier by installing it via npm: `npm i -D prettier` ([documentation for installing prettier as a package](https://prettier.io/docs/en/install)). To format the files, you need to use this command in a terminal opened in the root folder: <br />`npx prettier . --write`.
```

Comment:
Just curious, is VS code plugin requires prettier package install for the project or not? 
Yes, it does. [From the docs](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode#prettier-resolution):

```
This extension will use prettier from your project's local dependencies (recommended).
```

Since you installed prettier as a dev dep, let's remove this line, wdyt?

The less is more :)

Reply:
It work correctly without local Prettier